docs/examples/PanelBasic.js

```diff
@@ -1,5 +1,5 @@
 const panelInstance = (
-  <Panel>
+  <Panel onClick={ () => alert('You have clicked on me') }>
```

Comment:
I looked a bit and I believe all other examples in our codebase don't use an inline event handler like that, but rather a class method.